docs/data_usage_guides/endpoint_parameters/humanitarian_needs_parameters.yaml

```diff
@@ -66,19 +66,29 @@
   Description: >
```

Comment:
Maybe you can but sector above reference period here? I'm kind of noticing a pattern that goes like: 
* resource id
* dimensions
* metrics
* reference period
* geography